service/history/api/respondworkflowtaskcompleted/api.go

```diff
@@ -247,6 +247,7 @@ func (handler *WorkflowTaskCompletedHandler) Invoke(
 		}
 	}()
 
+	// TODO: this check is only
```

Comment:
finish this line?

Reply:
oops this sneaked in from an unrelated work. removed the line.